cmd/notation/registry.go

```diff
@@ -100,24 +68,15 @@ func getRemoteRepositoryForSign(ctx context.Context, opts *SecureFlagOpts, refer
 		return nil, err
 	}
 
-	// Notation enforces the following two paths during Sign process:
-	// 1. OCI artifact manifest uses the Referrers API.
-	// Reference: https://github.com/opencontainers/distribution-spec/blob/v1.1.0-rc1/spec.md#listing-referrers
-	// 2. OCI image manifest uses the Referrers API and automatically fallback
-	// 	  to Referrers Tag Schema if Referrers API is not supported.
-	// Reference: https://github.com/opencontainers/distribution-spec/blob/v1.1.0-rc1/spec.md#referrers-tag-schema
-	if !ociImageManifest {
-		logger.Info("Use OCI artifact manifest to store signature")
-		// ping Referrers API
-		if err := pingReferrersAPI(ctx, remoteRepo); err != nil {
+	if experimental.IsDisabled() || !allowReferrersAPI {
```

Comment:
Can we change it to `!experimental.IsDisabled() && allowReferrersAPI` and swap the `if` and `else` code block for easier if condition understanding?